features/thamos_advise.feature

```diff
@@ -22,6 +22,10 @@ Feature: Running thamos advise against deployment
 
         Examples: Advise
             | git_repo                                                             | runtime_environment     | user_stack  | static_analysis |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download_dataset        | without     | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download-dataset        | with        | without         |
             | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | training                | without     | with            |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | test_model              | with        | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | test_model              | with        | with            |
```

Comment:
`test-model`, also below

Reply:
I've added all entries with static analysis turned off to support jupyterlab-requirements. Some are kept turned on to test static analysis submission to the backand.